keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -38,11 +70,9 @@ This KEP proposes a possible solution for this, with the primary goal being to e
 
 ### Non-Goals
 
-We will expose the machinery and tools to bind a metric's labels to a discrete set of values.
+We will expose the machinery and tools to bind a metric's labels to a discrete set of values. The allowlist will be ingested via a new-added component metric flag.
 
-It is *not a goal* to implement and plumb this solution for each Kubernetes component (there are many SIGs and a number of verticals, which may have their own preferred way of doing things). As such it will be up to component owners to leverage this functionality that we provide, by feeding configuration data through whatever mechanism deemed appropriate (i.e. command line flags or reading from a file).
-
-These flags are really only meant to be used as escape hatches, and should not be used to have extremely customized kubernetes setups where our existing dashboards and alerting rule definitions are just not going to apply generally anymore.
```

Comment:
I don't think we need to delete all of this context.

Reply:
This part should be outdated because using a flag to ingest allow-list will be applied to all components and it is not a `non-goal`. Which part of these lines do you think I can keep?

Reply:
Actually this part is significantly easier for us to implement now that the CLI flag code is moved into component-base, so we can actually just do this.

Reply:
> As such it will be up to component owners to leverage this functionality that we provide, by feeding configuration data through whatever mechanism deemed appropriate (i.e. command line flags or reading from a file).

This is being deleted. Are we taking on the responsibility to hook this up for component owners now? If yes, and we don't have a goal of defining the allow-list, whose responsibility is that?

Reply:
I think this kep will provide the mechanism to ingest the metric allowlist with CLI flag. But as for the specific content of the allowlist, the component owners should define it.